docs/integration/js-example.md

```diff
@@ -1,9 +1,154 @@
+
+### Dockerfile for your server
+
+As every server is slightly different and we need to run them all, you need to provide us with a Dockerfile. If you haven't used Docker before, that's not a problem. Just use the Dockerfile from [our NodeJS SDK](https://github.com/UltimateTournament/ArcadeNodeJSSDK/blob/main/Dockerfile) and adapt the `npm run build` command to match your project.
+
+Just place this file next to your `package.json` and include it in all deliveries of your game to us.
+
+### Add our SDKs
+
+On the frontend run `npm i --save @ultimatearcade/client-sdk` and on the backend run `npm i --save @ultimatearcade/server-sdk`.
+
+BTW, you can use a single code base for frontend and backend, but you need to ship us a folder with static assets (HTML,JS,CSS,...) for the frontend as we don't run the frontend as a nodeJS server to save resources.
+
+### Initialize the SDK
+
+This should be the very first thing you do. You don't want your players to start playing only to be kicked a bit later
+because something wasn't quite right.
+
+#### On the server
+
+```typescript
+import { getSDK } from '@ultimatearcade/server-sdk'
+const uaSDK = getSDK()
+// this only resolves when the game server is actually needed (e.g. a player clicked on the game)
+// which can be hours after the process has started
+const ss = await uaSDK.getServerStatus()
+console.log("game allocated: ", ss)
+await yourServer.listen({ port: process.env.PORT }) // ensure to listen on the provided port
+```
+
+#### On the frontend
+
+```typescript
+import {getSDK} from '@ultimatearcade/client-sdk'
+const uaSDK = getSDK()
+const si = await uaSDK.getSessionInfo()
+// this is the address at which you'll reach your backend server, e.g. `your.host.com:1234`
+const serverAddress = si.server_address
+// this is a secret string that authenticates the user. Send it to your backend so it can use it later
+const = si.player_token
+```
+
+### Ensuring the connected player is valid
+
+When you get a connection (probably via WebSockets) you still need to ensure that a legitimate user has reached your server.
+
+The frontend should therefore send you the `player_token` it got above inside the very first message to your server. You then can verify it and get more player information like this:
+
+```typescript
+const playerInfo = await uaSDK.activatePlayer(initialMessage.player_token)
+const playerName = actResp.display_name
+const playerID = actResp.player_id
+```
+
+You MUST use the playerID from this call to identify the player to ensure that the same player can't connect twice. We try to prevent this on our end, too, but there are always some small time windows when this might happen.
+
+You should also not call this method twice. So it's a good idea to store the `player_token` in your player object, and if you get a connection check if it's just a reconnect after a dropped connection by comparing the player tokens.
+
+### Starting the game, locking the pool
```

Comment:
```suggestion
### (Pooled games) Starting the game
```

We might also want to show how to do it for leaderboard games, maybe another tutorial